tests/text/test_wer.py

```diff
@@ -1,83 +1,63 @@
 import pytest
-
-from torchmetrics.utilities.imports import _JIWER_AVAILABLE
-
-if _JIWER_AVAILABLE:
-    from jiwer import compute_measures
```

Comment:
I think we are fine to keep `jiwer` as a testing reference, see for example how other does it #443